spec/ruby/core/method/shared/eql.rb

```diff
@@ -54,6 +54,16 @@
     @m_foo.send(@method, m2).should be_true
   end
 
+  it "returns false if comparing a method defined via define_method and def" do
+    MethodSpecs::Methods.send :define_method, :defined_foo, lambda {}
```

Comment:
True, just used it because it was used in the above spec example :)
I'll change it